lab_3_keywords_textrank/main.py

```diff
@@ -152,7 +165,14 @@ def decode(self, encoded_tokens: tuple[int, ...]) -> Optional[tuple[str, ...]]:
                 sequence of string tokens
         In case of out-of-dictionary input data, None is returned
         """
-        pass
+        tokens = []
+        for token in encoded_tokens:
+            if token not in self._id2word:
+                return None
+            for number in self._id2word:
```

Comment:
why don't you just do `tokens.append(self._id2word[number])` ? Why do you need this loop?

Reply:
?